src/signTxOutput.c

```diff
@@ -367,6 +367,27 @@ static void signTxOutput_handleAssetGroup_ui_runStep()
 	UI_STEP_END(HANDLE_ASSET_GROUP_STEP_INVALID);
 }
 
+static int signTxOutput_lexiCompareViews(read_view_t lhs, read_view_t rhs)
```

Comment:
I think this could perhaps be moved to `signTxUtils.c` (or somewhere else) as it is a pretty generic function.

Reply:
It is generic, but I'm not sure where else it would be used, and I thought to keep it as local as possible until the need arises. But I would be interested in Mazák's opinion on this.

Reply:
What is the advantage of using read_views here instead of passing buffer + its size directly? (You can pass a wrong buffer size, but it's the same with the view, you can create a wrong one.)

Reply:
AFAIK we will need this for withdrawals and token minting, too, and in principle, it is a comparison between two cbor map keys, so I would consider putting it in `cbor.h` and naming it accordingly (perhaps not as a general comparison, but just a function returning bool iff the keys are in the correct order).

Reply:
> What is the advantage of using read_views here instead of passing buffer + its size directly? (You can pass a wrong buffer size, but it's the same with the view, you can create a wrong one.)

What is the advantage anywhere?

Reply:
For read views, it's mostly easier parsing of bip44 paths because those have variable length (same with address params). Especially when several of those are coupled together (e.g. wireBuffer containing two paths).